### PR TITLE
Less jumpy map systems with inline static info

### LINKED ISSUE #607: Less jumpy map systems with inline static info
Hey,

my wh-corp decided to ditch our old mapping system _J-Sig_ which we wrote years ago in favor of your great pathfinder as a solid modern framework for many more years of hunting in j-space :) 

After a few days we found the map to be a bit too busy for our taste with its animations, constantly moving systems when someone travels through and (sometimes buggy) active pilot tooltip. We feel that with a fixed system width and height in combination with a move of the active pilot list into a popover, we are able to "analyze" the map at a quick glance way easier.
In addition we wanted to provide our scouts and scanners with important information, so I added the statics to the map and added signature strength in their popover.
Since we like the grid snapping to keep everything tidy but are quite distracted by the grid-line-visuals we lowered there opacity a bit.

Preview:
preview

How do you feel about these changes? Any chance that you will pull them into the main line (with some modifications perhaps?) or are we "doomed" to keep using a fork? :)

### PR DESCRIPTION
**Pull request by tralafiti**
resolves #607

### CODE REVIEW

**Comment by WildStrawberryEVE** [comment]
I have to say I really like your ideas.

**Comment by tralafiti** [comment]
Added two small fixes to the PR

**Comment by dkrotil** [comment]
When will be this merged ? Looks very promising

**Comment by tralafiti** [comment]
@dkrotil I'm currently running some final test and plan to release a copy of 1.3.4 with my modifications - you will be able to switch between modified frontend and original js/css with one config setting - tomorrow

**Comment by WildStrawberryEVE** [comment]
Could you post a quick instruction - how to download it with git for people who are not that familiar with it?

**Comment by tralafiti** [comment]
@WildStrawberryEVE If you have a running instance of pathfinder you can follow the steps I wrote on the release page. Ping me on Slack if you need anything else

File: export/csv/wormhole.csv [hunk 1]
```diff
@@ -1,81 +1,81 @@
...
-"83";"Q063";"H";"500000000";"20000000";;"16";
\ No newline at end of file
+"Id";"Name";"Security";"MassTotal";"MassIndividual";"MassRegeneration";"MaxStableTime";"SignatureStrength";
+"1";"A009";"L";"500000000";"20000000";;"16";;
+"2";"A239";"L";"2000000000";"300000000";;"24";"5";
+"3";"A641";"H";"2000000000";"1000000000";;"16";"10";
+"4";"A982";"C6";"3000000000";"300000000";;"24";"2.22";
+"5";"B041";"C6";"3000000000";"300000000";"500000000";"48";;
+"6";"B274";"H";"2000000000";"300000000";;"24";"10";
+"7";"B449";"H";"2000000000";"1000000000";;"16";;
```
Comment (exodus4d) [inline export/csv/wormhole.csv]: According to http://wiki.eve-inspiracy.com/index.php?title=Wormhole_Signature_Strength_List **B449** has a strength of 2.5%

File: export/csv/wormhole.csv [hunk 2]
```diff
@@ -1,81 +1,81 @@
...
+"47";"O477";"C3";"2000000000";"300000000";;"16";"5";
+"48";"O883";"C3";"1000000000";"20000000";;"16";"5";
+"49";"P060";"C1";"500000000";"20000000";;"16";"5";
+"50";"Q003";"0.0";;"5000000";"1";"16";;
+"51";"Q317";"C1";"500000000";"20000000";;"16";"2.5";
+"52";"R051";"L";"3000000000";"1000000000";;"16";"5";
+"53";"R474";"C6";"3000000000";"300000000";;"24";"2.22";
+"54";"R943";"C2";"750000000";"300000000";;"16";"6.67";
+"55";"S047";"H";"3000000000";"300000000";;"24";;
+"56";"S199";"0.0";"3000000000";"1350000000";;"24";;
```
Comment (exodus4d) [inline export/csv/wormhole.csv]: According to http://wiki.eve-inspiracy.com/index.php?title=Wormhole_Signature_Strength_List **S199** has a strength of 10%

**Comment by exodus4d** [comment]
So I´m currently going through your PR and checking what changes have been made - great work so far! - Please forgive me, that I can´t accept a large PR with various changes and new feature  :( 
So I´m currently picking the features I want to add one by one into my develop branch. 
I´ll also create new "feature" Issues for each feature I took from your branch.

@tralafiti For now I have some questions regarding some code changes:
1. You changed som `MassTotal` values from `5000000000` to `3000000000`. Where can I verify that information? E.g. **C248** should have a `MassTotal` of `5000000000`
    - see http://evemaps.dotlan.net/wormholes
    - see http://wiki.eve-inspiracy.com/index.php?title=Wormhole_Signature_Strength_List
2. The missing `Signature strength` data for some WH types, is there any way to figure out the strength of the WHs that don´t have a SigStrength?

**Comment by exodus4d** [comment]
@tralafiti @WildStrawberryEVE I think 50% of the features from this branch are now on `develop` branch. But there are still some very useful features I want to implement now.

1. Showing the static WHs direct on map is awesome, I definitely want this. But there are some concerns about the way @tralafiti had implemented them:
    - Double the height of **any** system (even if it is not a WH) just for showing the static information is a big trade of. I´m a _fan of_ keeping everything _small_ and just take UI space if necessary. I´m juggling with the idea of showing them in _one line_ next to the system name.  So if no statics found (e.g. k-Space  system) no space is taken.
2. Moving **all** active pilots, in a system, to a tool tip will  hide some information from map, that is currently available. I like to see "at last" one pilot name on any system on map that has active pilots, it makes communication much easier on Voice-Com if you can ask a that guy something. Maybe showing one guy and move the rest to a tool tip would work...

**Comment by tralafiti** [comment]
@exodus4d Yeah, of course. Thanks for having a look :) I replied in a more detailed fashion on your Slack channel, but jut to have your questions also answered publicy I will write a shot reply here, too:

* I think the mass changes in the csv-export are caused by stale data in the old export. According to the sql dump and http://www.ellatha.com/eve/wormholelistview.asp?key=Wormhole+C248 the `MassTotal` of `3000000000` is the current value for a C248.
* Sig strength is mainly a tool to quickly find statics in a system. You can do a "overview" scan with one probe and the resulting signal strength group up on different levels (depending on your skills and ship of course). If you know the system has only one static which is in the highest band (e.g. H296) you can quickly scan the few signatures in this band to find the static without scanning all signatures. Of course this doesn't bring you any advantages for wandering WHs, since you don't know of there existence in this system beforehand. So, the "important" strength are documented :)

**Pull request closed without merging**